src/blockTypes/nixosConfigurations.nix

```diff
@@ -17,7 +29,51 @@
     type = "nixosConfiguration";
     # nixosGenerator's actions?
     # microvm action?
-    # nixos-rebuild action?
+    actions = {
+      currentSystem,
+      fragment,
+      fragmentRelPath,
+      target,
+      inputs,
+    }: let
+      pkgs = inputs.nixpkgs.${currentSystem};
+      getString = o: (l.elemAt (l.splitString ["/"] fragmentRelPath) o);
+      host = (getString 0) + "-" + (getString 2);
+      dc = getString 1;
+      bin = ''
+        bin=$(nix build .#${dc}.${host}.system --no-link --print-out-paths)/sw/bin
```

Comment:
Is `.system` a typo for `.config.system.build.toplevel` here? Or am I missing something that would alias to the shorter version?